### PR TITLE
Additions for a lot of the issues people have had

### PR DESCRIPTION
**Pull request by Dr-N0**
New: Event details are not immediately apparent
New: Mobile Format is difficult to use - Light fix, it'll look better in the next go around but it works for now and doesn't look horrible
Fix: Clicking on links in the mobile version doesn't happen anymore which allows for people to see the popovers
Fix: Times shown next to events
Fix: Load times kept at same levels despite adding double the elements

### CODE REVIEW

File: static/js/script.js [hunk 1]
```diff
@@ -12,13 +14,57 @@ $(document).ready(function() {
...
+        }
+    }
 });
 
 function updateInfo(x){
+    try{
+        var objectRow = x.closest('td').className;
+        var objectRowP = "";
+        var dayName = " ~ ";
+        if (objectRow == "col0") {
```
Comment (sgreene570) [inline static/js/script.js]: comparing rows to column names? Maybe have a table of col# -> day string?

File: static/js/script.js [hunk 2]
```diff
@@ -159,4 +207,31 @@ $(function(){
...
+
+function resetEvents(){
+    $('.smallTable > .eventLine').each(function () {
+        var obj = $(this);
+        obj.remove();
+    });
+}
+
+
+
```
Comment (sgreene570) [inline static/js/script.js]: extra whitespace

File: main.js [hunk 3]
```diff
@@ -64,8 +65,11 @@ function displayCalendar(auth, m, y) {
                             gDes = event.description
                         }
                     }
+                    if(gMin == "0"){
```
Comment (sgreene570) [inline main.js]: whats this for?

File: static/js/script.js (hunk 2 above)
Comment (Dr-N0) [reply static/js/script.js]: 👌

File: main.js (hunk 3 above)
Comment (Dr-N0) [reply main.js]: It defaults to 0 not 00, looks bad

File: static/js/script.js (hunk 1 above)
Comment (Dr-N0) [reply static/js/script.js]: You mean a variable for "col"?

Comment (sgreene570) [reply static/js/script.js]: well, sure, but what's objectRow more importantly?

File: static/js/script.js [hunk 4]
```diff
@@ -159,4 +207,27 @@ $(function(){
...
+        });
+    }
+}
+
+function resetEvents(){
+    $('.smallTable > .eventLine').each(function () {
+        var obj = $(this);
+        obj.remove();
+    });
+}
```
Comment (sgreene570) [inline static/js/script.js]: no newline

Comment (Dr-N0) [reply static/js/script.js]: done

File: static/js/script.js (hunk 1 above)
Comment (Dr-N0) [reply static/js/script.js]: It's not something I'll ever change to any other text, but I'll add it when I lint everything.

**Pull request merged**